### DISCUSSION TITLE
Creating a file or directory in long paths hitting character limit and cursor jumps back to root path ?

### DISCUSSION THREAD

**Discussion by devspectral** [stevearc/dressing.nvim#178] [Q&A]
Hi! When creating a file/directory (using "a" key and also happening in renaming a file/folder) with long nested paths, for some reason it is hitting the character limit and will just jump or delete the entire path in the "Create file" box. See screenshot below for reference:

![image](https://github.com/user-attachments/assets/075b64e7-ba89-4733-a88b-45daff5c1f43)

Then when it hits the limit (end of the create file bar), it jumps to this.

![image](https://github.com/user-attachments/assets/3c407d4b-5891-4b4b-bcb4-7a9727c3d266)

Any thoughts on how to expand the limit? or better yet how to only show the current working directory and not the entire path so that I won't hit that character limit?

Thank you and I appreciate all the help!

**Comment by stevearc** [comment]
There is no character limit that I'm aware of; this is probably due to some interaction with something else in your config. If you go through the bug report process it has some steps for putting together a minimal repro. That will either help me reproduce the issue on my machine so I can help, or you will realize that the issue lies elsewhere.
